app/controllers/socure_webhook_controller.rb

```diff
@@ -41,4 +45,21 @@ def verify_queue(authorization_header:)
       )
     end
   end
+
+  def log_webhook_receipt
+    event = socure_params[:event]
+    return if event.blank?
+
+    analytics.idv_doc_auth_socure_webhook_received(
+      created_at: event[:created],
+      customer_user_id: event[:customerUserId],
+      event_type: event[:eventType],
+      reference_id: event[:referenceId],
+      user_id: event[:customerUserId],
```

Comment:
why are we logging `customerUserId` twice with different names?

Reply:
@zachmargolis - our uuid is passed back from Socure in the`customerUserId` field and we're logging what Socure is sending us ...
but logging the `customerUserId` out as `user_id` allows us to include this event in CW when querying the  events.log with properties.user_id='[uuid]' for all activities for a user

Reply:
Should we be sending the internal UUID?

Reply:
yes, we'd be passing the uuid as a unique identifier which would be returned to us ... in order for us to to know  which user is taking the acctions ...  sounds, like this is of concern? 🤔 

it looks like the latest version of the API, which we are implementing, now includes `docVTransactionToken` which would allow us a means to accomplish our goals without passing Socure the uuid in the CaptureApp request 🤔

Reply:
yeah, I think we generally do not send internal UUIDs externally and generate a new identifier as needed.

Reply:
[we do pass internal UUIDs to vendors](https://github.com/18F/identity-idp/blob/c1d5d0bcfad1b0b88cdbdf417a70351455bab3e2/app/services/proofing/lexis_nexis/request.rb#L79-L88)

Reply:
> @zachmargolis - our uuid is passed back from Socure in the`customerUserId` field and we're logging what Socure is sending us ... but logging the `customerUserId` out as `user_id` allows us to include this event in CW when querying the events.log with properties.user_id='[uuid]' for all activities for a user

That makes sense! I'm saying if we do the remapping to `user_id` then there is no reason to log `customer_user_id` as well because it's the exact same value with a different name, right?

Reply:
yes, your point is valid as it is a duplicative value ... i think the intent is to reflect what/how it's received while also being searchable 🤔

Reply:
yes, you're point is valid 👍🏿 and logging `customer_user_id` as `user_id` appear to be a a simple key transformation/renaming ... i think there is value in logging the fields _exactly_ as received by the vendor may serve us with any future troubleshooting and provide certainty in what we're receiving and that we have not altered it in any way.